osCam/userconfig/migrations/0001_initial.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.0.4 on 2022-04-18 02:55
+
+from django.conf import settings
```

Comment:
I don't know why this migrations file is in here, I thought we had a git ignore for these?

Reply:
I thought so to, either way im deleting it